fili-core/src/main/java/com/yahoo/bard/webservice/util/StreamUtils.java

```diff
@@ -199,4 +199,18 @@
     public static <T> Set<T> setMerge(Set<T> a, Set<T> b) {
         return Stream.concat(a.stream(), b.stream()).collect(Collectors.toSet());
     }
+
+    /**
+     * Merge two sets without modifying either, preserving order.
+     * This method implements {@link BinaryOperator} for use in stream reductions.
+     *
+     * @param a  One input set
+     * @param b  Another input set
+     * @param <T>  The type of the underlying sets
+     *
+     * @return  A new set containing the values of the original sets
```

Comment:
single space between tag and description. We don't capitalize the first letter in `return` description

Reply:
I'm getting to the point where if the tools don't enforce it where we might want to consider dropping this style requirement.  If the tools don't support it, it's frustrating for engineers to police it.